disaster-recovery/backup/Dockerfile

```diff
@@ -2,8 +2,8 @@ FROM python:3.12-alpine3.20
 
 COPY requirements.txt /requirements.txt
 RUN apk update && apk upgrade
-RUN pip install --upgrade pip
-RUN pip install -r /requirements.txt
+RUN pip install --no-cache-dir --upgrade pip
+RUN pip install --no-cache-dir -r /requirements.txt
 RUN pip uninstall -y wheel setuptools
 COPY ./cross_account_backup.py /cross_account_backup.py
 RUN chmod 755 cross_account_backup.py
```

Comment:
Would it make sense to use the same absolute path /cross_account_backup.py in the RUN command?

Reply:
Yes I think so